cardano-config/src/Cardano/Tracing/ToObjectOrphans.hs

```diff
@@ -961,3 +983,205 @@ instance (Show (GenTx blk), Show (GenTxId blk))
         , "txsNoLongerValidRemoved" .= String (pack $ show $ txs1)
         , "mempoolSize" .= String (pack $ show $ mpSz)
         ]
+
+instance ( StandardHash blk
+         , ToObject (LedgerError blk)
+         , ToObject (ValidationErr (BlockProtocol blk)))
+      => ToObject (ChainDB.InvalidBlockReason blk) where
+  toObject verb (ChainDB.ValidationError extvalerr) =
+    mkObject
+        [ "kind"  .= String "ValidationError"
+        , "error" .= toObject verb extvalerr
+        ]
+  toObject verb (ChainDB.InChainAfterInvalidBlock point extvalerr) =
+    mkObject
+        [ "kind"  .= String "InChainAfterInvalidBlock"
+        , "point" .= String (pack $ show point)
```

Comment:
There's also a function in this module `showPoint`. Not sure if that would be appropriate or if this output is somehow nicer.